pkg/awsutils/awsutils.go

```diff
@@ -1356,7 +1368,7 @@ func (cache *EC2InstanceMetadataCache) DescribeAllENIs() (DescribeAllENIsResult,
 		if interfaceType == "trunk" {
 			trunkENI = eniID
 		}
-		if interfaceType == "efa" {
+		if interfaceType == "efa" || interfaceType == "efa-only" {
```

Comment:
eventually we should move the interfaceType to some enum for trunk, efa, efa-only..

Reply:
Makes sense @jayanthvn. I will add a backlog task for this